data/campaigns/World_Conquest/lua/map/postgeneration/2A_Springs.lua

```diff
@@ -155,7 +155,7 @@ function world_conquest_tek_map_decoration_2a()
 	while #terrain_to_change > 0 and mathx.random(10) ~= 1 do
 		local loc = terrain_to_change[mathx.random(#terrain_to_change)]
 		map[loc] = "^Efm"
-		local terrain_to_change = wct_store_possible_flowers("G*^Fet")
+		terrain_to_change = wct_store_possible_flowers("G*^Fet")
```

Comment:
This looks like a bugfix, I think it will change the behavior and needs testing.

Reply:
What would be the test case here?  The only difference now is that it will exit the loop early if there's no terrain left to change rather than waiting for `mathx.random(10)` to return 1.

Reply:
Ah yeah, didn't realise how little it could change.